fixtures/get-bindings-proxy/tests/index.test.ts

```diff
@@ -44,35 +46,61 @@ describe("getBindingsProxy", () => {
 		await Promise.allSettled(devWorkers.map((i) => i.stop()));
 	});
 
-	it("correctly obtains var bindings", async () => {
-		const { bindings, dispose } = await getBindingsProxy<Bindings>({
-			configPath: wranglerTomlFilePath,
+	describe("var bindings", () => {
+		it("correctly obtains var bindings from both wrangler.toml and .dev.vars", async () => {
+			const { bindings, dispose } = await getBindingsProxy<Bindings>({
+				configPath: wranglerTomlFilePath,
+			});
+			const { MY_VAR, MY_JSON_VAR, MY_DEV_VAR } = bindings;
+			expect(MY_VAR).toEqual("my-var-value");
+			expect(MY_JSON_VAR).toEqual({
+				test: true,
+			});
+			expect(MY_DEV_VAR).toEqual("my-dev-var-value");
+			await dispose();
 		});
-		const { MY_VAR, MY_JSON_VAR } = bindings;
-		expect(MY_VAR).toEqual("my-var-value");
-		expect(MY_JSON_VAR).toEqual({
-			test: true,
+
+		it("correctly makes vars from .dev.vars override the ones in wrangler.toml", async () => {
+			const { bindings, dispose } = await getBindingsProxy<Bindings>({
+				configPath: wranglerTomlFilePath,
+			});
+			const { MY_VAR_A } = bindings;
+			expect(MY_VAR_A).not.toEqual("my-var-a");
```

Comment:
this check is completely unnecessary, but I like it as it makes the test clearer, but if preferred I can remove it 🙂

Reply:
I like this as an example of looking ahead to how the test could fail.

The test could fail for not matching "my-dev-var-a" but in that case it's very likely to not match because the value is "my-var-a" and this assertion catches that case specifically. It's kind of clear from the test name but I might also help the dev looking at the failing test by putting a comment after this assertion:

```suggestion
			expect(MY_VAR_A).not.toEqual("my-var-a"); // if this fails, the value was read from wrangler.toml – not .dev.vars
```